command.go

```diff
@@ -783,7 +782,7 @@ func (c *Command) initHelpCmd() {
 			Run: func(c *Command, args []string) {
 				cmd, _, e := c.Root().Find(args)
 				if cmd == nil || e != nil {
-					c.Printf("Unknown help topic %#q.", args)
+					c.Printf("Unknown help topic %q\n", args)
```

Comment:
Blech, I made a mistype in the first review. I wanted you to use `\n` and I mistyped `%%q\n`. Which now I realize is why you used `%%`.

How about `%#q\n`?  That work?